src/test/java/com/coinbase/exchange/api/orders/OrderTests.java

```diff
@@ -102,10 +102,29 @@ public void getAllOpenOrders() {
     }
 
     @Test
-    public void getFills() {
-        List<Fill> fills = orderService.getAllFills();
+    public void getFillsByProductId() {
+        List<Fill> fills = orderService.getFillsByProductId("BTC-USD", 100);
         assertTrue(fills.size() >= 0);
     }
+
+    @Test
+    public void createMarketOrderBuyThenGetFillByOrderId() {
```

Comment:
naming should follow format of `should do something When X happens`

Reply:
shouldGetFilledByOrderIdWhenMakingMarketOrderBuy()

Reply:
Ahh...  Ok, method has been renamed as suggested.